### DISCUSSION TITLE
Insight: Burn up chart

### DISCUSSION THREAD

**Discussion by uelshioura** [community/community#77499] [Projects and Issues]
I want to display a burnup chart grouped by status.
However, when the x-axis is set to time, the glue option becomes invisible.

![image](https://github.com/community/community/assets/117799025/36edadf7-dfdf-4292-b697-b7cc769982d2)

![image](https://github.com/community/community/assets/117799025/2eca3968-1ae5-4b65-81f6-f8312f755bc4)

Are there any additional settings required?

**Comment by colinmurphy01** [comment]
I don't even a `time` option available available to me?

![Screenshot 2023-12-20 at 2 48 38 PM](https://github.com/community/community/assets/24654605/5455fff5-dea4-4bff-bb02-fa7318851d6c)

**Comment by uelshioura** [comment]
Thanks for your reply.
Maybe you're not a Team or Enterprise user.

`Historical charts are available with GitHub Team and GitHub Enterprise Cloud for organizations.`
https://docs.github.com/en/enterprise-cloud@latest/issues/planning-and-tracking-with-projects/viewing-insights-from-your-project/about-insights-for-projects
